README.md

```diff
@@ -6,7 +6,7 @@ What is TogetherJS?
 
 TogetherJS is a service for your website that makes it surprisingly easy to collaborate in real-time.
 
-Using TogetherJS two people can interact on the same page, seeing each other's cursors, edits, and browsing a site together.  The TogetherJS service is included by the web site owner, and a web site can customize and configure aspects of TogetherJS's behavior on the site.
+Using TogetherJS two people can interact on the same page, seeing each other's cursors, edits, and browsing a site together,at the same time.  The TogetherJS service is included by the web site owner, and a website can customize and configure aspects of TogetherJS's behavior on the site.
```

Comment:
Doesn't `browsing a site together` imply it's being done at the same time? Also lacks a space after comma.